docs/wiki/99 - Developers/SQL Dictionary.md

```diff
@@ -9,17 +9,41 @@
-	| `Study_site` | Does this site recruit candidates?             | This field will affect the appearance of the center in dropdowns in several modules of LORIS |
+	| `Account`    |                                                  | Deprecated                                                                                   |
+	| `Study_site` | Does this site recruit candidates?               | This field will affect the appearance of the center in dropdowns in several modules of LORIS |
+
+- Table: `visit`
+
+ 	*This table stores the list of timepoints configured for the study.*
+ 	
+ 	|     Field    |          Description          |                                    Notes                                    |
+	|:------------:|:------------------------------------------------:|:--------------------------------------------------------:|
+	| `VisitID`   | Identifier of the Timepoint    | Avoid setting this field explicitly when inserting data, it auto increments.|
+	| `VisitName` | Back end name of the Timepoint | This name is used in file and imaging uploads                               |
+
+- Table: `Visit_Windows`
+
+ 	*This table associates timepoints with optimal age ranges for quality control monitoring.*
+ 	
+ 	|        Field         |                                        Description                                        |                         Notes                          |
+	|:--------------------:|:-----------------------------------------------------------------------------------------:|:------------------------------------------------------:|
+	| `ID `                | Identifier of the entry                                                                                   | Avoid setting this field explicitly when inserting data, it auto increments.           |
+	| `Visit_label `       | Name of the visit                                                                                         |  |
+	| `WindowMinDays `     | Candidate's minimum age in days for visit to be flagged as _within **permitted** parameters of the study_ | The only effect of this field is a YES/NO flag showing up on the instrument_list page|
+	| `WindowMaxDays `     | Candidate's maximum age in days for visit to be flagged as _within **permitted** parameters of the study_ | The only effect of this field is a YES/NO flag showing up on the instrument_list page|
+	| `OptimumMinDays `    | Candidate's minimum age in days for visit to be flagged as _within **optimal** parameters of the study_   | The only effect of this field is a YES/NO flag showing up on the instrument_list page|
+	| `OptimumMaxDays `    | Candidate's maximum age in days for visit to be flagged as _within **optimal** parameters of the study_   | The only effect of this field is a YES/NO flag showing up on the instrument_list page|
```

Comment:
I don't understand, what is the difference between `WindowMinDays` and `OptimumMinDays`? According to the description of both, they do the same thing?

Same question for Max days.

Reply:
yes, the only difference is one gets a "optimal" flag, the other "permitted" flag

@driusan how wrong am i ?

Reply:
"Permitted" means it needs to be in that range (ie. if it's not it shouldn't be in the study). "Optimal" means it should be in that range (and if it's not, there's a warning on the instrument.) They're used for the Window Difference on an instrument, with things outside "Permitted" being highlighted in red as an error.

Reply:
@driusan but as far as the software is concerned, they are just flags, there is no real effect of being within permitted or optimal right ?